### PR TITLE
Basic project setup

### LINKED ISSUE #1: Basic project setup
- Create basic project setup
- Should be able to route between pages
- Should show navbar and sidebar
- Add the required npm packages

### PR DESCRIPTION
**Pull request by ttahm3d**
PR for Issue #1 

- Created folders for various files
- Added all the required packages
- Added routing between components

Preview
localhost_3000_

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,145 @@
...
+}
+
+.main-container {
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  gap: 0.5rem;
+  min-height: 100vh;
+}
+
+/* @media screen and (min-width: 80em) {
```
Comment (aishwarya-bm) [inline src/index.css:136]: remove comments if not required

File: src/router/Router.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Route, Routes } from "react-router-dom";
+import { Suspense, lazy } from "react";
+
+const LandingPage = lazy(() => import("../pages/landing-page/LandingPage"));
+const ApiResponse = lazy(() => import("../pages/api-response/ApiResponse"));
+const Explore = lazy(() => import("../pages/explore/Explore"));
+const History = lazy(() => import("../pages/history/History"));
+const Playlist = lazy(() => import("../pages/playlist/Playlist"));
+const WatchLater = lazy(() => import("../pages/watch-later/WatchLater"));
+const Liked = lazy(() => import("../pages/liked-videos/LikedVideos"));
```
Comment (aishwarya-bm) [inline src/router/Router.jsx:10]: I like how lazy loading is handled here. I am also going to do this.

**Review by aishwarya-bm — APPROVED** [review]
Good folder structure and clean code with extension points. I liked the lazy loading of scripts.

File: src/index.css (hunk 1 above)
Comment (ttahm3d) [reply src/index.css:136]: Have updated code now. There were a couple of ideas of handling header and sidebar... will remove later.

File: src/router/Router.jsx (hunk 2 above)
Comment (ttahm3d) [reply src/router/Router.jsx:10]: There are couple of things that you need to take care of. You need to export the component as default to lazy load it.

File: src/components/Header/Header.module.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+.menu {
+  display: none;
+}
+
+.menu__logo {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  align-items: center;
```
Comment (Sameony) [inline src/components/Header/Header.module.css:26]: repeated property

**Review by Sameony — APPROVED** [review]
Loved the way of toggling the sidebar and the concise and modular commit messages. Great code.

**Pull request merged**